unblob/handlers/archive/tar.py

```diff
@@ -31,6 +31,7 @@ def _get_tar_end_offset(file: io.BufferedIOBase):
     last_member = tf.getmembers()[-1]
     last_file_size = BLOCK_SIZE * (1 + (last_member.size // BLOCK_SIZE))
     end_offset = last_member.offset + HEADER_SIZE + last_file_size + END_BLOCK_SIZE
```

Comment:
`last_offset` would be a better name, or name the padded offset `padded_end_offset` or `end_offset_padded` instead.

Reply:
Fixed.